test/autorest/urlgroup/queries_test.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package urlgrouptest
+
+import (
+	"context"
+	"encoding/base64"
+	"generatortests/autorest/generated/urlgroup"
+	"net/http"
+	"testing"
+)
+
+func getQueriesClient(t *testing.T) urlgroup.QueriesOperations {
+	client, err := urlgroup.NewDefaultClient(nil)
+	if err != nil {
+		t.Fatalf("failed to create enum client: %v", err)
+	}
+	return client.QueriesOperations()
+}
+
+func TestArrayStringCsvValid(t *testing.T) {
+	client := getQueriesClient(t)
+	result, err := client.ArrayStringCsvValid(context.Background(), []string{
+		"ArrayQuery1", "begin!*'();:@ &=+$,/?#[]end", "", "",
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if result.StatusCode != http.StatusOK {
```

Comment:
This is just a question, why did you switch from using the deep equals func to just checking the status code? It's the same result for the successful status codes, so it doesnt really matter, but just wondering why you preferred this.

Reply:
It was shorter to write since there is no response body to compare, just the status code.